Modules/Sources/Support/UI/Bot Conversations/ConversationListView.swift

```diff
@@ -0,0 +1,248 @@
+            }
+
+            guard case .none = oldValue else {
+                preconditionFailure("You cannot add a substate – one already exists")
+            }
+
+            return .loaded(conversations, newValue)
+        }
+
+        func updateSubstate(_ newValue: ViewSubstate) -> Self {
+            guard case .loaded(let conversations, let oldValue) = self else {
+                preconditionFailure("You cannot transition to a substate unless the current state is `loaded`")
+            }
+
+            guard oldValue != nil else {
+                preconditionFailure("You cannot update to a new substate – none exists")
+            }
+
+            return .loaded(conversations, newValue)
+        }
+
+        func clearSubstate() -> Self {
+            guard case .loaded(let conversations, let oldValue) = self else {
+                preconditionFailure("You cannot clear substate unless the current state is `loaded`")
+            }
+
+            guard oldValue != nil else {
+                preconditionFailure("You cannot clear substate – none exists")
+            }
+
+            return .loaded(conversations, nil)
+        }
+
+        var isPartiallyLoaded: Bool {
+            guard case .partiallyLoaded = self else {
+                return false
+            }
+
+            return true
+        }
+    }
+
+    enum ViewSubstate {
+        case deletingConversations(Task<Void, Never>)
+        case deletingConversationsError(Error)
+    }
+
+    @EnvironmentObject
+    private var dataProvider: SupportDataProvider
+
+    @State
+    var state: ViewState = .loading
+
+    @State
+    var selectedConversations = Set<String>()
+
+    private let currentUser: SupportUser
+
+    public init(currentUser: SupportUser) {
+        self.currentUser = currentUser
+    }
+
+    public var body: some View {
+        VStack {
+            switch self.state {
+            case .loading:
+                ProgressView("Loading Bot Conversations")
+            case .partiallyLoaded(let conversations): self.conversationList(conversations)
+            case .loaded(let conversations, _): self.conversationList(conversations)
+            case .loadingConversationsError(let error):
+                ErrorView(
+                    title: "Unable to load conversations",
+                    message: error.localizedDescription
+                )
+            }
+        }
+        .navigationTitle("Conversations")
+        .toolbar {
+            ToolbarItem(placement: .primaryAction) {
+                NavigationLink {
+                    ConversationView(
+                        conversation: nil,
+                        currentUser: currentUser
+                    ).environmentObject(dataProvider)
+                }
+                label: {
+                    Image(systemName: "square.and.pencil")
+                }
+                .disabled(!currentUser.permissions.contains(.createChatConversation))
```

Comment:
This button is disabled for my test accounts. Not sure why...

Reply:
I'd guess it's because they don't have a paid WP.com plan?